IntegrationTests/PurchaseTester/PurchaseTester/Info.plist

```diff
@@ -34,6 +34,10 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>LSApplicationCategoryType</key>
+	<string></string>
+	<key>UIApplicationSupportsMultipleScenes</key>
+	<true/>
```

Comment:
this is unrelated, but it enables multi-tasking support for `PurchaseTester` on iPad, which is useful for some testing some stuff (like the manage subscriptions modal)